### ISSUE TITLE
Ability to set failed state on test case which has multiple auto tests

### ISSUE THREAD

**Issue by Gennadiii** [testomatio/app#927]
**Is your feature request related to a problem? Please describe.**
I have 1 test case in testomat which results in 1 automated test suite. This test suite has multiple tests which all have the same test case id from testomat. When first test fails and the second one passed I have green result in testomat when indeed playwright has a failed test.

**Describe the solution you'd like**
Ability to set via configuration to mark a test case as failed if it failed at least once during automated test run

**Describe alternatives you've considered**
Alternative is one - not use testomat reports

Basically it should look like negative strategy when merging runs. Also this report shows retries because multiple tests ran under the same id. You might consider taking test (it) name to distinguish if it's a retry or another test.

**Comment by poliarush** [comment]
@DavertMik are we going to implement this?
